CSharp/Library/Microsoft.Bot.Connector.Teams.Shared/CardExtensions.cs

```diff
@@ -53,5 +53,33 @@ public static Attachment ToAttachment(this O365ConnectorCard card)
                 ContentType = O365ConnectorCard.ContentType
             };
         }
+
+        /// <summary>
+        /// Creates a new attachment from <see cref="FileInfoCard"/>.
+        /// </summary>
+        /// <param name="card"> The instance of <see cref="FileInfoCard"/>.</param>
+        /// <returns> The generated attachment.</returns>
+        public static Attachment ToAttachment(this FileInfoCard card)
```

Comment:
The class description is for O365 and we are putting files card here?

Reply:
Changing description.